source/sdk/Iterators.ooc

```diff
@@ -79,13 +79,13 @@ _MappingIterator: class <T, S> extends Iterator<S> {
 		this _backend free()
 		super()
 	}
-	hasNext?: func -> Bool {
+	hasNext?: override func -> Bool {
```

Comment:
For a future version of rock, it'd be nice to be able to remove the question mark for this function (and likewise for `hasPrev?`)